api/mesh/options.proto

```diff
@@ -22,9 +22,12 @@ message KumaResourceOptions {
   // Whether to skip type registration for this resource.
   bool skip_registration = 6;
 
-  KumaKdsOptions kds = 10;
```

Comment:
Can we change the type of existing field in proto? Won't it break anything?  https://protobuf.dev/best-practices/dos-donts/#:~:text=Don't%20Change%20the%20Type%20of%20a%20Field,-Almost%20never%20change&text=The%20protobuf%20docs%20outline%20a,superset%20of%20the%20old%20one.

Reply:
yes exactly it's not shared so it's fine